### PR TITLE
Budget App

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
In this pull request, I built a mobile web application where you can manage your budget: you have a list of transactions associated with a category so that you can see how much money you spent and on what.

I created a Ruby on Rails application that allows the user to:

-   register and log in, so that the data is private to them.
-   introduce new transactions associated with a category.

#### link to the presentation 
[https://www.loom.com/share/dc0eb1ed6c434fdc854c11a618a790b2](https://www.loom.com/share/dc0eb1ed6c434fdc854c11a618a790b2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,237 @@
...
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (arslanbisharat) [inline README.md:229]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

File: test/channels/application_cable/connection_test.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class ApplicationCable::ConnectionTest < ActionCable::Connection::TestCase
```
Comment (arslanbisharat) [inline test/channels/application_cable/connection_test.rb:3]: - [ ] Please make sure the app has unit tests for all the most important components of the RoR application.

File: README.md [hunk 3]
```diff
@@ -1 +1,237 @@
-# budget-app
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
```
Comment (arslanbisharat) [inline README.md:3]: - [ ] In the recorded video, kindly address a piece of code or feature that you liked the most.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Owusu-Desmond, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Owusu-Desmond, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggested changes.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**